lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1184,7 +1184,7 @@ public function send($content = null) {
 			}
 			CakeLog::write(
 				$config['level'],
-				PHP_EOL . $contents['headers'] . PHP_EOL . $contents['message'],
+				PHP_EOL . $contents['headers'] . PHP_EOL . PHP_EOL . $contents['message'],
```

Comment:
> Also separate log headers and body with empty line.

Email log looks better with empty line between headers and body:
```
Subject: the title
From: CakePHP <cake@cakephp.org>
the body
```

vs 

```
Subject: the title
From: CakePHP <cake@cakephp.org>

the body
```

Then you can easily copy to `.eml` file and preview email in any email client without manually inserting new line.